Makefile.vbcc

```diff
@@ -7,15 +7,16 @@ VASM=vasmm68k_mot
 VLINK=vlink
 LDFLAGS=-L$(NDK)/linker_libs -lamiga -lauto
 CONFIG=+aos68k
-RM=rm
+ddRM=rm
 
 TESTAPP=ast_tests
 ERRORHOST=error_host
 
-SOURCE_FILES = ObjectMemory.o ObjectMemory_Storage.o ObjectMemory_FreeList.o ObjectMemory_Allocation.o ObjectMemory_RefCounting.o ObjectMemory_GC.o RealWordMemory.o Interpreter.o Interpreter_Error.o
-HEADER_FILES = ObjectMemory.h ObjectMemory_Constants.h ObjectMemory_Storage.h ObjectMemory_FreeList.h ObjectMemory_Allocation.h ObjectMemory_RefCounting.h ObjectMemory_GC.h RealWordMemory.h Types.h Interpreter.h
-HEADER_FILES2 = Interpreter_Constants.h Interpreter_Error.h
-#had to split this because the line was too long
+OM_SOURCEFILES =  ObjectMemory.o ObjectMemory_Storage.o ObjectMemory_FreeList.o ObjectMemory_Allocation.o ObjectMemory_RefCounting.o ObjectMemory_GC.o RealWordMemory.o
+INTERP_SOURCEFILES =  Interpreter.o Interpreter_Error.o
+
+OM_HEADERFILES = ObjectMemory.h ObjectMemory_Constants.h ObjectMemory_Storage.h ObjectMemory_FreeList.h ObjectMemory_Allocation.h ObjectMemory_RefCounting.h ObjectMemory_GC.h RealWordMemory.h 
+INTERP_HEADERFILES = Types.h Interpreter.h Interpreter_Constants.h Interpreter_Error.h
```

Comment:
This might be a PITA for line length but `Types.h` is used in the `ObjectMemory`.

Reply:
I noticed that, but its not like the build separate targets.  I can pull types.h into a SHARED_HEADERFILES section :-)

Reply:
fair enough :)